<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCurvature</class>
 <widget class="QWidget" name="QtCurvature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtCurvature</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spin_size">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Index:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spin_index">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Select Id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_outer">
          <property name="text">
           <string>Outer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_inner">
          <property name="text">
           <string>Inner</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_showImage">
        <property name="text">
         <string>Show Contour</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_clearolddata">
        <property name="text">
         <string>Clear Old Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_savedata">
        <property name="text">
         <string>Save Data</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="check_passed">
          <property name="text">
           <string>Passed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_failed">
          <property name="text">
           <string>Failed</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
